Controllers/ReviewController.cs

```diff
@@ -97,6 +97,91 @@ public IHttpActionResult GetAllReviews()
             return Ok(reviews);
         }
 
+
+        // ===================================================================================================
+        [HttpGet]
+        [Route("~/api/user/{userId:int}/reviews")]
+        public IHttpActionResult GetUserReviews(int userId)
+        {
+            Review review = null;
+
+            try
+            {
+                SqlConnection sqlConnection = new SqlConnection(DB_CONNECTION_STRING);
+                using (sqlConnection)
+                {
+                    SqlCommand selectReviewById = new SqlCommand(@"SELECT * FROM Review
```

Comment:
Give names of column instead of "*" for better query performance:
SELECT [Rev_ID]
      ,[Rev_Date]
      ,[Rev_Rate]
      ,[Rev_Recommendation]
      ,[Rev_Comment]
      ,[Rev_Usefull]
      ,[Rev_Title]
      ,[Use_ID]
      ,[Acc_ID]
  FROM [dbo].[Review]

Reply:
good catch